sdk/src/account_info.rs

```diff
@@ -0,0 +1,304 @@
+use std::{cell::RefCell, rc::Rc};
+
+use anchor_lang::prelude::Result;
+use solana_program::pubkey::Pubkey;
+
+use crate::{
+    account_meta::LightAccountMeta,
+    address::PackedNewAddressParams,
+    compressed_account::{
+        CompressedAccount, CompressedAccountData, OutputCompressedAccountWithPackedContext,
+        PackedCompressedAccountWithMerkleContext,
+    },
+    error::LightSdkError,
+    merkle_context::PackedMerkleContext,
+};
+
+/// Information about compressed account which is being initialized.
+#[derive(Debug)]
+pub struct LightInputAccountInfo<'a> {
+    /// Lamports.
+    pub lamports: Option<u64>,
+    /// Address.
+    pub address: Option<[u8; 32]>,
+    /// Account data.
+    pub data: Option<&'a [u8]>,
+    /// Merkle tree context.
+    pub merkle_context: PackedMerkleContext,
+    /// Root index.
+    pub root_index: u16,
+}
+
+/// Information about compressed account which is being mutated.
+#[derive(Debug)]
+pub struct LightAccountInfo<'a> {
+    /// Input account.
+    pub(crate) input: Option<LightInputAccountInfo<'a>>,
+    /// Owner of the account.
+    ///
+    /// Defaults to the program ID.
+    pub owner: Option<Pubkey>,
```

Comment:
can we make this the same as in [Solana account info](https://docs.rs/solana-program/latest/solana_program/sysvar/slot_history/struct.AccountInfo.html) so that we don't copy more bytes than necessary?
```suggestion
    pub owner:  &'a Pubkey>,
```